messages.py

```diff
@@ -0,0 +1,23 @@
+class Messages:
+    WELCOME = """
+        🤖 Welcome to this Instagram feed bot! (sorry for the uncool name)\n
```

Comment:
You know OOP is not my strong suit, so it may sound like a stupid question: this class is ultimately a storage for different constants. I remember once you told me about dataclasses. Is this related/Should this be one? And how is this an advantage over a big dict:

```
messages = {
    "not_implemented": "blablabla"
    "new_post": "blablabla"
}
```

Reply:
`dataclass` is usually used to store the state of the object, or just some data. They allows you to save some space and improve readability but there is a convention that defines the usage of dataclasses more as a storage, as schema if you wish.
Here is an example of defining the same class:
```python
@dataclass
class Person:
    name: str
    age: int
```
is the same as 
```python
class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age
 
    def __repr__(self):
        return f"Person(name={self.name!r}, age={self.age!r}"
     
    def __eq__(self, other):
        if other.__class__ is self.__class__:
            return (self.name, self.age) == (other.name, other.age)
        return NotImplemented
```

Talking about the `dict` or the `class` - I would say that most of time it's the matter of taste. Reading a big `dict` is also a little bit pain in the ass. In your example you will have something like:
```python
messages.get("welcome", "").format(...)
```
Note that you will always need to return an empty string, just to be safe.
Also using a class has a huge advantage - you can define methods or other attributes or anything else you need to work with your messages.
For example, instead of using the construction from my PR
```python
Messages.WELCOME.format(
            MAX_PROFILES_PER_USER=self.database.MAX_PROFILES_PER_USER,
            DEFAULT_UPDATE_INTERVAL_HOURS=self.database.DEFAULT_UPDATE_INTERVAL_HOURS
        )
```
you can define a method in `Messages` class that will make you message definition a little bit more readable:
```python
def get_message(self, message_name: str, *args, **kwargs) -> str:
    return getattr(self, message_name).format(*args, **kwargs)
```
and use it in your code like this:
```python
Messages().get_message(
        "WELCOME",
        MAX_PROFILES_PER_USER=5,
        DEFAULT_UPDATE_INTERVAL_HOURS=4
    )
```

The main thing of all this changes is that when you are writing code in a company or "for business" uses - you need to be sure, that this code will be easy to understand and easy to maintain, not only that "it works" :-)